src/dbnode/generated/proto/namespace/namespace.proto

```diff
@@ -33,6 +33,31 @@ message NamespaceOptions {
     bool coldWritesEnabled                          = 10;
     NamespaceRuntimeOptions runtimeOptions          = 11;
     google.protobuf.BoolValue cacheBlocksOnRetrieve = 12;
+    AggregationOptions aggregationOptions           = 13;
+}
+
+message AggregationOptions {
+    repeated Aggregation aggregations = 1;
+}
+
+message Aggregation {
+    AggregationAttributes attributes    = 1;
+    DownsampleOptions downsampleOptions = 2;
+}
+
+message AggregationAttributes {
+    int64 resolutionNanos = 1;
```

Comment:
Consider making `google.protobuf.Duration`

Reply:
All the other time fields in the `NamespaceOptions` are nanos so will leave as is for consistency.

Reply:
👍